platform/darwin/test/MGLCodingTests.m

```diff
@@ -80,6 +80,32 @@ - (void)testPolyline {
     XCTAssertEqual([multiLine coordinate].latitude, multiLineCenter.latitude);
     XCTAssertEqual([multiLine coordinate].longitude, multiLineCenter.longitude);
     
+    CLLocationCoordinate2D segmentCoordinates[] = {
+        CLLocationCoordinate2DMake(35.040390, -85.311477),
+        CLLocationCoordinate2DMake(35.040390, -85.209510),
```

Comment:
We should vary both the latitude and the longitude to ensure that the logic works in both directions.